reference/commands.rst

```diff
@@ -75,6 +75,36 @@ and these :ref:`custom command examples <examples_extensions_custom_commands>`
 - :doc:`conan upload <commands/upload>`: Upload packages from the local cache to a specified remote
 
 
+.. _commands_output:
+
+Commands Output to stdout and stderr
+------------------------------------
```

Comment:
```suggestion
**Commands Output to stdout and stderr**
```

It looks to me that the ``Formatters`` section is a bit weird, I think I prefer all subsections in this page to look similar and at the same level:
- Consumer commands
- Creator commands
- Commands output
- Formatters

Most likely a taste issue, so I am fine merging as-is